roles/commcare_analytics/templates/superset/celery_beat_start_script.j2

```diff
@@ -12,5 +12,6 @@ echo "Starting {{ app_name }}_celery_beat as `whoami`"
 # (do not use --daemon).
 exec celery --app superset.tasks.celery_app:app  \
     beat \
+    -s /home/ubuntu/www/run/celerybeat-schedule \
```

Comment:
Beat failed to start with error accessing celerybeat-schedule file.
It was not clear where this file was though a suggestion was to explicitly pass the path to the file
https://docs.celeryq.dev/en/stable/userguide/periodic-tasks.html#starting-the-scheduler

This file is where it would store tasks info for itself.